operator/v1/types_ingress.go

```diff
@@ -125,6 +127,13 @@ type IngressControllerSpec struct {
 	//
 	// +optional
 	NodePlacement *NodePlacement `json:"nodePlacement,omitempty"`
+
+	// tlsSecurityProfile specifies settings for TLS connections for ingresscontrollers.
+	//
+	// If unset, the default is based on the ingresses.config.openshift.io/cluster resource.
+	//
+	// +optional
+	TLSSecurityProfile *configv1.TLSSecurityProfile `json:"tlsSecurityProfile,omitempty"`
```

Comment:
Is `spec.tlsSecurityProfile` mutable?

Reply:
Maybe given we go out of our way to call out things which are immutable, we can assume that it's mutable? I think generally users expect to be able to change spec fields and have those changes reflected.